src/openMVG/localization/CCTagLocalizer.cpp

```diff
@@ -25,7 +23,9 @@ CCTagLocalizer::CCTagLocalizer(const std::string &sfmFilePath,
                                const std::string &descriptorsFolder)
 {
   using namespace openMVG::features;
-  
+
+  cctag::initCCTagDetection();
```

Comment:
Is this something related to just the GPU version or something general, that must be used for both CPU and GPU version? Otherwise i think we need some `#if`